linux/ibus-keyman/tests/scripts/test-helper.inc.sh

```diff
@@ -250,7 +250,7 @@ function _setup_ibus() {
   #shellcheck disable=SC2086
   ibus-daemon ${ARG_VERBOSE-} --daemonize --panel=disable --address=unix:abstract="${TEMP_DATA_DIR}/test-ibus" ${IBUS_CONFIG-} &> /tmp/ibus-daemon.log
   PID=$(pgrep -f "${TEMP_DATA_DIR}/test-ibus")
-  echo "kill -9 ${PID} || true" >> "$CLEANUP_FILE"
+  echo "if kill -9 ${PID}; then ibus restart || ibus start; fi || true # ibus-daemon" >> "$CLEANUP_FILE"
```

Comment:
```suggestion
  echo "if kill -9 ${PID}; then ibus restart || ibus start; fi # ibus-daemon" >> "$CLEANUP_FILE"
```

The `|| true` shouldn't be required now that we are doing `kill` within the `if` clause. Unless you meant:

```suggestion
  echo "if kill -9 ${PID}; then ibus restart || ibus start || true; fi # ibus-daemon" >> "$CLEANUP_FILE"
```


An alternative might also be something like this:

```suggestion
  echo "kill -9 ${PID} && (ibus restart || ibus start) || true # ibus-daemon" >> "$CLEANUP_FILE"
```